schema/changelog-4.4b.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd"
+  logicalFilePath="changelog-4.4b">
+
+  <changeSet author="author" id="changelog-4.4b-all">
+    <sql>
```

Comment:
Why is it SQL instead of proper liquibase definition?

Reply:
Fixed. Was my first time working with liquibase